Mention this document updates ICESDP to allow ".local" ICE candidates (#129)
rtcweb-wg/mdns-ice-candidates: draft-ietf-rtcweb-mdns-ice-candidates.md

Description {#description}
======================

This document extends the {{ICESDP}} specification to allow the generation

> and resolution of ICE candidates that use mDNS FQDN names.

Reviewer: We don't have a definiton of what an mDNS FQDN name is. How about "a specific subset of FQDN addresses"?  "FQDN addresses" is used instead of "FQDN names" to be consistent with ICESDP and to also avoid the repetition of  "name".

> and resolution of ICE candidates that use mDNS FQDN names.

Author: ```suggestion
and resolution of ICE candidates that use a specific subset of FQDN addresses.
```

Section 5.1 of {{ICESDP}} states that 'An agent generating local candidates
MUST NOT use FQDN addresses. An agent processing remote candidates MUST
ignore candidate lines that include candidates with FQDN or IP address
versions that are not supported or recognized.'.
This specification updates these sentences as follows: 'An agent generating

> local candidates MUST NOT use FQDN addresses which top-level domain name

Reviewer: nit
```suggestion
local candidates MUST NOT use FQDN addresses of which the top-level domain name
```

is not ".local". An agent processing remote candidate MUST ignore candidate

> lines that include candidates with FQDN which top-level domain name is not

Reviewer: nit
```suggestion
lines that include candidates with FQDN of which the top-level domain name is not
```
